### PR TITLE
Add Check Status and update for Docker image with Spark

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-graphene/README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ SGX-based Trusted Big Data ML allows user to run end to end big data analytics a
 Before run the following command, please modify the pathes in the build-docker-image.sh file at first. <br>
 Then build docker image by running this command: <br>
 ```bash
+chmod a+x build-docker-image.sh
+chmod a+x Dockerfile
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/scala/docker-graphene/README.md:12]: chmod dockerfile is not necessary

Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-graphene/README.md]: tks，done~

File: ppml/trusted-big-data-ml/scala/docker-graphene/check-status.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    if [ $? -eq 1 ] ; then
+        echo "Cannot find master log at" $MASTERLOG 
+    else 
+        MASTERSUCCESS=$(cat $MASTERLOG | grep "I have been elected leader")
+        if [ -z "$MASTERSUCCESS" ] ; then
+            echo "Master initilization failed. See" $MASTERLOG " for details."
+            echo "To restart Master, run ./start-spark-standalone-master-sgx.sh in the docker container."
+        fi
+    fi
+    MASTERPORT=$(netstat -nlp | grep 8080)
```
Comment (ManfeiBai) [inline ppml/trusted-big-data-ml/scala/docker-graphene/check-status.sh:21]: container no command

Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-graphene/check-status.sh:21]: graphene not initialized in new container now to run check-status

Comment (ManfeiBai) [reply ppml/trusted-big-data-ml/scala/docker-graphene/check-status.sh:20]: Done~

**Review by glorysdj — APPROVED** [review]
LGTM

**Review by glorysdj — COMMENTED** [review]
there are still some issues.

**Pull request merged**